packages/core/src/pagination/pagination-button.element.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { baseStyles, CdsBaseButton, property, querySlot } from '@cds/core/internal';
+import { html } from 'lit-element';
+import { styles } from './pagination-button.element.css.js';
+
+export type CdsPaginationButtonAction = 'prev' | 'next' | 'first' | 'last';
+
+export class CdsPaginationButton extends CdsBaseButton {
```

Comment:
https://github.com/vmware/clarity/blob/0300acd8e1c40b7b480e400271ce3306c74cb568/packages/core/src/button/button.element.ts#L35-L63 

The comment will help the storybook page to show the available css props